.github/workflows/netlify.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Deploy to Netlify
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: |
+            ~/.npm
+            **/node_modules
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Build OpenAPI redoc, Typescript types, and client library docs
+        run: npm run build:docs && mv ./docs/.tmp docs-build/
```

Comment:
Can you add `docs-build/` to .gitignore?

Reply:
The `docs-build` dir only exists temporarily within this github action's execution because the next step will not upload anything from a dot directory (i.e. `.tmp/`)